### PR TITLE
Dev ems allyn

### PR DESCRIPTION
**Pull request by allyncubero**
- adding of employee to scheduling sheet(aside from the members of the groups)
- delete function of scheduling sheet
- reports on EMS(Leave and Pass Slip reports)

### CODE REVIEW

File: apps/employee-monitoring/src/components/modal/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx [hunk 1]
```diff
@@ -163,10 +146,10 @@ const AddFieldSsModal: FunctionComponent<AddFieldSsModalProps> = ({
   });
 
   // on submit
-  const onSubmit = async (data: any) => {
+  const onSubmit = async () => {
     // extract the unnecessary items for posting
-    const { scheduleName, customGroupName, id, customGroupId, ...rest } =
-      currentScheduleSheet;
+    const { scheduleName, customGroupName, id, customGroupId, ...rest } = currentScheduleSheet;
+    console.log(rest);
```
Comment (toyaaponesto) [inline apps/employee-monitoring/src/components/modal/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx:152]: remove later

**Review by toyaaponesto — COMMENTED** [review]
All goods

**Review by toyaaponesto — APPROVED** [review]
All goods

File: apps/employee-monitoring/src/components/modal/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx (hunk 1 above)
Comment (allyncubero) [reply apps/employee-monitoring/src/components/modal/monitoring/scheduling-sheet/field/AddFieldSsModal.tsx:152]: Will remove after the BE fixes the adding of employees to scheduling sheet

**Pull request merged**